pulpcore/cli/core/context.py

```diff
@@ -227,6 +227,13 @@ class PulpContentRedirectContentGuardContext(PulpEntityContext):
     ID_PREFIX = "contentguards_core_content_redirect"
 
 
+class PulpDistributionContext(PulpEntityContext):
```

Comment:
Should we have all the Distributions inherit from this? (I know there is little value in this unless we have some fancy extra methods or NULLABLES here.)

Reply:
I would say we leave it as is now and in the future we find extra bits that need to be shared we move it to common and have others inherit.